aikau/src/main/resources/alfresco/core/topics.js

```diff
@@ -72,6 +72,30 @@ define([],function() {
       CLEAR_SELECTED_ITEMS: "ALF_CLEAR_SELECTED_ITEMS",
 
       /**
+       * This can be published to make a request to create a dialog. It is typically handled by the
+       * [DialogService]{@link module:alfresco/services/DialogService}.
+       *
+       * @instance
+       * @type {string}
```

Comment:
Should these be defined with @event (as per http://dev.alfresco.com/resource/docs/aikau-jsdoc/_AlfDocumentListTopicMixin.js.html#line156 ) so they show up in the events view in JSDoc & can be referenced in @listens and @fires annotations?